dp3t-sdk/sdk/src/main/java/org/dpppt/android/sdk/internal/database/KnownCases.java

```diff
@@ -11,20 +11,20 @@
 
 	String ID = "id";
 	String ONSET = "onset";
-	String BUCKET_DAY = "day";
+	String BUCKET_TIME = "day";
```

Comment:
iOS hat das in der DB auf batchTimestamp umbenennt. wäre cool wenn beide Datenbanken das gleiche Schema hätten.